tools/ci-operator-prowgen/Gopkg.toml

```diff
@@ -3,5 +3,7 @@ required = [
 	"github.com/openshift/ci-operator/pkg/api",
 	"k8s.io/test-infra/prow/config",
 	"k8s.io/test-infra/prow/kube",
-	"k8s.io/api/core/v1"
+	"k8s.io/api/core/v1",
```

Comment:
I'm confused why these are necessary.  I would not expect 5 million lines of vendor dependencies to be pulled in just for one test case.